src/waves/waves.js

```diff
@@ -34,7 +34,9 @@ export class MdWaves {
     }
 
     this.attributeManager.addClasses(classes);
-    Waves.attach(this.element);
+    if (!(typeof window.require === 'function' && typeof window.require.specified === 'function' && window.require.specified('waves'))) {
```

Comment:
Any idea why this works at all with elements added after `DOMContentLoaded`? I still don't get it (had a look at the fork's changes already).